### PR TITLE
(GH-9) Added GitterStats

### PR DESCRIPTION
**Pull request by gep13**

### CODE REVIEW

**Comment by gep13** [comment]
Relates to #9

File: GitterStats/run.ps1 [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+"Begin request $(Get-Date -Format 'yyyy-MM-dd HH:mm:ss')"
+
+$headers = @{
+    'Content-Type' = 'application\json'
+    'Accept' = 'application\json'
+    'Authorization' = "Bearer $env:GITTER_TOKEN"
+}
+
+# TODO: This group id needs to be brought out as a configuration parameter
+$packageData = Invoke-RestMethod -Uri "https://api.gitter.im/v1/groups/57542ccdc43b8c6019776af0/rooms" -ErrorAction SilentlyContinue -Headers $headers `
```
Comment (devlead) [inline GitterStats/run.ps1]: Should this be an array of one to many groups?

**Comment by gep13** [comment]
@devlead let me know what you think about the changes.

File: GitterStats/run.ps1 [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+                    'Exported'      = [System.DateTimeOffset]::Now
+                    'Name'          = $_.name
+                    'UserCount'     = $_.userCount
+                    'Favourite'     = $_.favourite
+                    'public'        = $_.public
+            }}
+
+    if ($gitterData -eq $null)
+    {
+        $gitterData = ("No package data found for gitter room")
```
Comment (devlead) [inline GitterStats/run.ps1]: No room data found?

File: GitterStats/run.ps1 [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+                    'Exported'      = [System.DateTimeOffset]::Now
+                    'Name'          = $_.name
+                    'UserCount'     = $_.userCount
+                    'Favourite'     = $_.favourite
+                    'public'        = $_.public
+            }}
+
+    if ($gitterData -eq $null)
+    {
+        $gitterData = ("No package data found for gitter group")
```
Comment (devlead) [inline GitterStats/run.ps1]: No group data found?

**Review by devlead — APPROVED** [review]
LGTM 👍  2 minor comments, but we can fix those later.

File: GitterStats/run.ps1 (hunk 3 above)
Comment (gep13) [reply GitterStats/run.ps1]: Doh! Copy/pasta...

File: GitterStats/run.ps1 (hunk 2 above)
Comment (gep13) [reply GitterStats/run.ps1]: Doh! Copy/pasta...

**Comment by gep13** [comment]
@devlead refresh page 😄

**Pull request merged**